Journal week 5

pope410211 opened the pull request.

pope410211 committed:
update

pope410211 committed:
clearning house

pope410211 committed:
updated .gitignore

pope410211 committed:
forgot to merge before update...

Merge branch 'remodeling' of github.com:pope410211/pope410211.github.io into Journal-week-5

pope410211 committed:
updating files

pope410211 committed:
Q's unaswered

pope410211 committed:
ignore spelling erro

pope410211 committed:
tutorial update

BFGriffith commented:
Nice scripts tutorial. CSS-tricks is indeed a great resource. Some good basics about merge-conflicts, but maybe link to more tools or commands regarding those. Good concise reflection on our halftime week... :+1:

gatorpazz commented:
You're reflective journal made me wanna vomit with joy but I'd recommend just a teensy bit longer. YOUR TUTORIAL WAS ROCKING!!! YOUR LINKED ARTICLE IS NOT A SPECIFIC ARTICLE BUT I'LL GIVE IT A PLUS 1 ANYWAYS!!!!! :+1: :+1: :+1:

pope410211 committed:
made adjustment

sjoyal commented:
@pope410211 you can be my boss for the right price fo' sho sir. Like the tutorial you included, there's some markdown issues with the code block, but that's an easy fix. Great job :+1:

pope410211 committed:
Update 2015-05-07-Tutorials-are-for-Xbox-people.md

pope410211 committed:
Update 2015-05-07-Tutorials-are-for-Xbox-people.md

pope410211 committed:
uping...?

pope410211 committed:
created merge, not sure how?

Merge branch 'Journal-week-5' of github.com:pope410211/pope410211.github.io into Journal-week-5
